packages/ember-data/lib/system/store.js

```diff
@@ -250,7 +252,8 @@ Store = Ember.Object.extend({
 
     if (DS.Adapter.detect(adapter)) {
```

Comment:
Well, I think I'm just too bad explaining what I have in mind :confounded: 
So I will try again.

Historically, the user can affect a class or a string to the `adapter` property. As a result, there is this chain of checks. If the `adapter` property is a string, then the adapter is looked up via the container, or if it's a class (actually a subclass of DS.Adapter), then it is instantiated here.

My question is: do we still want to allow the user define the `adapter` property as a class. It seems to me this would be inconsistent with the "string everywhere" paradigm.

Please, tell me it's clear now... or tell me to take some intensive English training :smile_cat: